### PR TITLE
Bump image versions in values.yml

### PR DESCRIPTION
**Pull request by brandboat**
**Problem:**
bump images in values.yml for v1.4 branch

**Solution:**
bump images in values.yml for v1.4 branch

**Related Issue:**
#6568 

**Test plan:**

### CODE REVIEW

File: deploy/charts/harvester/values.yaml [hunk 1]
```diff
@@ -374,7 +374,7 @@ harvester-node-disk-manager:
     repository: rancher/harvester-node-disk-manager
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: v0.7.2
+    tag: v0.7.3
```
Comment (Vicente-Cheng) [inline deploy/charts/harvester/values.yaml]: Hi @brandboat,
Sorry for that, but we need to bump to v0.7.4 (image tag) for NDM

Comment (brandboat) [reply deploy/charts/harvester/values.yaml]: Thanks, addressed in [dfde93d](https://github.com/harvester/harvester/pull/6654/commits/dfde93d493e2de9170087821ed60b8aa19b9175c)

Comment (Vicente-Cheng) [reply deploy/charts/harvester/values.yaml]: Thank you!

**Comment by brandboat** [comment]
Waiting for the upcoming release of the network-controller-harvester.

**Review by bk201 — APPROVED** [review]

**Review by bk201 — APPROVED** [review]

**Review by Vicente-Cheng — APPROVED** [review]
thanks!

**Pull request merged**